tags/TagMapContext.md

```diff
@@ -0,0 +1,144 @@
+# Summary
+A `Tag` is a key-value pair that can be used to label anything that is associated
+with a specific operation, such as an HTTP request. These `Tag`s are used to
+aggregate measurements in a [`View`](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/DataAggregation.md#view) 
+according to unique value of the `Tag`s. The `Tag`s can also be used to filter (include/exclude)
+measurements in a `View`.
+
+# TagMap 
+`TagMap` is an abstract data type that represents collection of tags. 
+i.e., each key is associated with exactly one value, but multiple keys can be
+associated with the same value.  `TagMap` is serializable, and it represents
+all of the information that could be propagated across process boundaries.
+
+## TagKey
+
+A string or string wrapper, that contains two parts, scope and key. The scope
+of a tag is used to determine if a tag should be propagated or not.
```

Comment:
scope is the extent of propagation right? it wouldn't make sense to never propagate even internal to process right?

Reply:
right and right. What I meant was at every propagation point scope is used to determine whether to propagate or not. But I will make it clear to not sound like it is a binary property (propagate or not-propagate) of a tag.